sql/moz-fx-data-shared-prod/stripe_external/promotion_codes_v1/init.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR REPLACE TABLE
+  `moz-fx-data-shared-prod`.stripe_external.promotion_codes_v1
+PARTITION BY
+  DATE(created)
+CLUSTER BY
+  created
+AS
+WITH events AS (
+  SELECT
+    created AS event_timestamp,
+    `data`.promotion_code.*,
+  FROM
+    events_v1
+  WHERE
+    `data`.promotion_code IS NOT NULL
+  UNION ALL
+  SELECT
+    created AS event_timestamp,
+    *,
+  FROM
+    initial_promotion_codes_v1
```

Comment:
This `initial_promotion_codes_v1` table doesn't exist yet.  Will it be created, and if so, how?

Reply:
it will be manually created by running something like `script/bqetl stripe import --before-date=2022-01-19 --api-key=REDACTED --resource=PromotionCode --table=moz-fx-data-shared-prod.stripe_external.initial_initial_promotion_codes_v1`